package com.restapi.RestApi.Dto;
import lombok.AllArgsConstructor;
import lombok.Data;


@Data
@AllArgsConstructor
public class StudentDto {

    private long id;
    private String name;
    private String email;



}
